packages/frontend/core/src/desktop/dialogs/setting/workspace-setting/new-workspace-setting-detail/members/index.tsx

```diff
@@ -0,0 +1,44 @@
+import { Button, Tooltip } from '@affine/component';
+import { SettingRow } from '@affine/component/setting-components';
+import { AffineErrorBoundary } from '@affine/core/components/affine/affine-error-boundary';
+import { useWorkspaceInfo } from '@affine/core/components/hooks/use-workspace-info';
+import { useI18n } from '@affine/i18n';
+import { useService, WorkspaceService } from '@toeverything/infra';
+import type { ReactElement } from 'react';
+
+import type { SettingState } from '../../../types';
+import { CloudWorkspaceMembersPanel } from './cloud-members-panel';
+import * as styles from './styles.css';
+
+export const MembersPanel = ({
+  onChangeSettingState,
+}: {
+  onChangeSettingState: (settingState: SettingState) => void;
+}): ReactElement | null => {
+  const workspace = useService(WorkspaceService).workspace;
+  const isTeam = useWorkspaceInfo(workspace.meta)?.isTeam;
```

Comment:
maybe use team subscription, useWorkspaceInfo just for workspace list

Reply:
Open another PR to handle this issue